packages/app/src/app/pages/common/Modals/PreferencesModal/Experiments/ContainerLSP.tsx

```diff
@@ -17,6 +17,8 @@ export const ContainerLSP: React.FunctionComponent = () => {
     setContainerLSP(val);
     window.localStorage.setItem('CONTAINER_LSP', val.toString());
     location.reload();
+
+    // TODO(@someone): call the API call on POST /api/v1/users/current with body `{ experiments: { container_lsp: true } }`
```

Comment:
I think we should revert @SaraVieira's changes from https://github.com/codesandbox/codesandbox-client/pull/3571/commits/6e8d7e68a732230fa755bb2259e3863b77b46406 and use the action again